newpr.py

```diff
@@ -205,6 +207,11 @@ def get_page_content(self, url):
         except urllib2.URLError:
             return None
 
+    def get_issues(self, labels):
```

Comment:
As written, this will hit problems with paging when there are high enough numbers of results. We should avoid this by performing a search for the particular filename, rather than retrieving all issues labelled intermittent and filtering client-side. We should reuse what crowbot does: https://github.com/servo/crowbot/commit/4cd902337c37bd236f9ac52aa030d9a247c9e493